### PR TITLE
Billi api integration for Task Data

### PR DESCRIPTION
**Pull request by chirag-compro**

### CODE REVIEW

File: client/app/_services/data.service.ts [hunk 1]
```diff
@@ -0,0 +1,93 @@
...
+    taskData.id = res.TaskFriendlyID,
+    taskData.app = this.getTaskApp(res.TaskFriendlyID),
+    taskData.testStatus = true,
+    taskData.commitStatus = true,
+    taskData.stepData = this.getScenario(res.ScenarioItemList);
+    taskData.previewUrl = "preview/"+res.TaskFriendlyID,
+    taskData.testUrl = "test/"+res.TaskFriendlyID
+    return taskData;
+  }
+  getTaskApp(taskid){
```
Comment (vsingb3) [inline client/app/_services/data.service.ts]: getAppName

File: client/app/_services/stepModel.ts [hunk 2]
```diff
@@ -0,0 +1,19 @@
+export class Step {
```
Comment (chirag-compro) [inline client/app/_services/stepModel.ts:1]: merge step details

Comment (chirag-compro) [reply client/app/_services/stepModel.ts:1]: also remove step name

File: client/app/_services/taskModel.ts [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+
+export class Task {
+ 		id:string;
+        app:string;
+        testStatus: boolean;
+        commitStatus:boolean;
+        // createdBy:UserX;
+        // lastModifiedBy:UserY;    
+        stepData: Array<Step>;
+        previewUrl:string;
```
Comment (chirag-compro) [inline client/app/_services/taskModel.ts]: remove preview/test url

File: client/app/_services/taskData.service.ts [hunk 4]
```diff
@@ -1,25 +1,46 @@
...
 import { Observable } from 'rxjs/Observable';
 import 'rxjs/add/operator/catch';
 import 'rxjs/add/operator/map';
+import 'rxjs/add/observable/of';
+
+import { Task } from './taskModel'
+import { Step } from './stepModel'
 
 @Injectable()
 export class DataService {
```
Comment (chirag-compro) [inline client/app/_services/taskData.service.ts]: taskDataService

**Review by vsingb3 — APPROVED** [review]

**Pull request merged**